packages/cursorless-vscode/package.json

```diff
@@ -838,7 +838,11 @@
           }
         },
         "cursorless.experimental.snippetsDir": {
-          "description": "Directory containing snippets for use in cursorless",
+          "description": "Directory containing snippets for use in Cursorless",
+          "type": "string"
+        },
+        "cursorless.experimental.hatShapesDir": {
```

Comment:
Can you have settings not contributed in package?

Reply:
Yeah if you just remove it from package.json everything will still work, it just won't appear on autocomplete schema. That's how we deprecate settings fwiw: we just remove them from package.json but keep checking for them in our code just in case

Reply:
If you want I can remove it from the package file?